2019-01-25 15:37:23,071 ERROR: Exception on /people_notes/2 [GET] [in /home/gene/.local/lib/python3.6/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/gene/.local/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/gene/.local/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/gene/.local/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/gene/Desktop/CRM/app/routes.py", line 103, in people_notes
    select_people = People.query.filter_by(user_account_pk=account_pk, id=people_id).first()
NameError: name 'account_pk' is not defined
2019-01-25 15:37:42,955 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:38:08,460 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:39:04,435 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:39:30,038 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:39:58,987 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:40:25,232 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:40:45,512 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:42:37,331 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:43:29,279 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:43:48,890 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
2019-01-25 15:44:21,558 INFO: ReDefined CRM Startup [in /home/gene/Desktop/CRM/app/__init__.py:53]
